lib/test/TestSuite.js

```diff
@@ -614,30 +622,30 @@ class TestSuite {
 
         const include = typeof includeRaw === "object" ? includeRaw : this.getTagsFromString(includeRaw);
         const exclude = typeof excludeRaw === "object" ? excludeRaw : this.getTagsFromString(excludeRaw);
+        if (include && include.length === 0 && exclude && exclude.length === 0) {
+            return;
+        }
+        
+        const suiteTags = typeof this.tags === "object" ? this.tags : this.getTagsFromString(this.tags);
```

Comment:
`typeof this.tags === "object"` also match with `null` or '[ ]', try using `this.tags.constructor == Object`

![image](https://user-images.githubusercontent.com/2177405/63976667-71c27880-ca77-11e9-9b1e-749182a61e62.png)